satellite/repair/repair_test.go

```diff
@@ -63,7 +63,7 @@ func TestDataRepair(t *testing.T) {
 		var (
 			testData         = testrand.Bytes(8 * memory.KiB)
 			minThreshold     = 3
-			successThreshold = 7
+			successThreshold = 6
```

Comment:
I don't much like changing this without knowing why it fixes the test. Can we at least run it by @mobyvb - looks like he made some related changes earlier today

Reply:
It should pass with either 6 or 7 - this change should be inconsequential, though I am still curious why it was made

Reply:
@mobyvb this test was failing on my branch with the following info: 
```
	Error Trace:	repair_test.go:94
            	            				run.go:70
            	Error:      	Should be false
            	Test:       	TestDataRepair/Sqlite
            	Messages:   	there is not enough available nodes for repairing: need= 8, have= 7
```
changing to 6 fixed it for some reason

Reply:
I know what it is! We changed the test to set `toKill := numPieces - toDisqualify - int(minReq)` where previously it was `toKill := numPieces - toDisqualify - int(minReq+1)`. This results in the edge case where the segment gets uploaded to 10 pieces, and we want to kill 6 nodes (10 pieces - 1 disqualified - 3 min = 6). Then, since we have 14 nodes total, if we kill 6, we have 7 nodes remaining. 4 of those will be holding pieces, and 1 of those 4 will be disqualified. This means we have only 3 remaining nodes to repair to, which is not sufficient.

Long story short, it's my (and @VinozzZ 's) fault, and the change @JessicaGreben is what we should do. Otherwise we can increase the total nodes to 15 or decrease max threshold to 9. I think I will make a quickfix PR for this since it will affect other people's stuff.

Reply:
found the issue with the test, fixed with this PR: https://github.com/storj/storj/pull/2973

Reply:
So do you still want to make this change, or leave it so you don’t get blamed when it breaks something 2 years from now haha

Reply:
lol! I was going to revert this change but then i would need to get approvals again. 😢 so getting blamed forever seems easier lol

Reply:
For what it's worth, I think 7 is the better value (since it means there are 2 nodes between repair/success threshold). But the getting approvals again thing is a real problem that I understand.

Reply:
ok ok i will change it back.

Reply:
_done_